tsdapiclient/tacl2.py

```diff
@@ -210,20 +260,18 @@ def cli(
     config_show,
     config_delete,
     session_delete,
-    register
+    register,
+    ignore_prefixes,
+    ignore_suffixes,
+    upload_cache_show,
+    upload_cache_delete,
+    upload_cache_delete_all,
+    cache_disable,
 ):
     """tacl2 - TSD API client."""
     token = None
     if verbose:
         os.environ['DEBUG'] = '1'
-    if env and not register:
```

Comment:
Didn't see notes on the removal of this check, but is the idea that this will be caught by the the `handle_request_errors()` decorator?

My main intention when adding it was to avoid making users input their credentials in states where the API is unavailable before telling them this, so perhaps it could be added under the `if auth_required` case instead.

Reply:
The `env` has a default value, so this branch is executed for "config" operations like checking your config, listing local cache, deleting it, and so on - which is incorrect: one does not need API connectivity to be able to do that. Since there is a branch for handling connection errors in the decorator, I think this is good enough.

Reply:
done